### DISCUSSION TITLE
Zoom Teleport/Warp Effect

### DISCUSSION THREAD

**Discussion by fleeasura123** [pixijs/pixijs#7534] [Q&A]
Good day! I am very fascinated by this effect. 

![zoom teleport warp](https://user-images.githubusercontent.com/53055963/121692414-a5f71400-cafa-11eb-817e-a2c8f23d2680.gif)

Any tips how can we implement it on PixiJS? 
Or demos I can see.. Or what should I learn to implement such effect?

Any help is appreciated.

**Comment by ivanpopelyshev** [comment]
If its webgl site, dig out the shader, maybe use SpectorJS to do that (capture frame in transition). Then make a pixi filter based on it

**Comment by fleeasura123** [comment]
> If its webgl site, dig out the shader, maybe use SpectorJS to do that (capture frame in transition). Then make a pixi filter based on it

Thank you for the tips! will research on that! nice!

**Comment by bigtimebuddy** [comment]
While not totally the same, but try out this demo using the [ZoomBlurFilter](https://filters.pixijs.download/main/docs/PIXI.filters.ZoomBlurFilter.html).

https://jsfiddle.net/bigtimebuddy/82zwq4om/

You might be able to apply this with BulgePinchFilter or Displacement filter to get more distortion.

**Reply by fleeasura123** [reply]
Wow! This is amazing! Thank you! I will experiment using that code 😉

**Reply by bigtimebuddy** [reply]
Also some interesting results with BulgePinch: https://jsfiddle.net/bigtimebuddy/bw54gjpo/

**Reply by fleeasura123** [reply]
Wow! Nice effect! Thank you!
